var/spack/repos/builtin/packages/py-checkm-genome/package.py

```diff
@@ -13,18 +13,26 @@ class PyCheckmGenome(PythonPackage):
     homepage = "https://ecogenomics.github.io/CheckM"
     pypi = "checkm-genome/checkm-genome-1.0.11.tar.gz"
 
+    version("1.2.1", sha256="33907aa7bbf029f8345e33df80d5c89b7a719041f55ece4f7470cd061c8eff76")
     version("1.0.13", sha256="ffb7e4966c0fac07c7e6e7db6f6eb5b48587fa83987f8a68efbaff2afb7da82e")
     version("1.0.11", sha256="e475d9817d12fa771dbccc80f47758b742fc67c25261dc8ca0c0dc898c2a5190")
 
     # pip silently replaces distutils with setuptools
+
```

Comment:
Looks like I forgot to note that I confirmed both new sha256.

Reply:
I usually put `when` before `type` just because that's the order they are defined in the directive definition. I also usually put deps in order from newest to oldest since that's the order we put versions. But both of these are preferences, not requirements. Wish we could teach black/flake8 to enforce this kind of ordering easily...